### ISSUE TITLE
Request for Updating Dataset Link Update and Clarification on Text-to-Image Process

### ISSUE THREAD

**Issue by Volta-lemon** [minghangz/ResVG#1]
Dear Researcher,Greetings!
I would like to express my sincere appreciation for the work you have done.
I am writing to kindly request that you reply this issue and provide an updated link for the dataset, which appears to be currently inactive. I would be grateful if you could prioritize this matter in your response.
Additionally, I am keen to understand the process by which you convert text to images and encode them with markers. I noticed that there is no mention of the Stable Diffusion  processed images in the dataset loading section in this code, nor any instructions on loading SD weights in the README. Could you please elucidate on when and how this text-to-image conversion is performed within your workflow? 
Your insights would be invaluable.Thank you very much for your time and assistance. I look forward to your reply.

**Comment by minghangz** [comment]
Thank you for your attention. We apologize for the broken dataset link. We have updated the link in the README: https://disk.pku.edu.cn/link/AA6D7224C7C18142BE9F5F6EAFC17813F3

Regarding the process of converting text to images, we used SDXL-Turbo to generate images based on text and utilized CLIP to encode the image features during the preprocessing stage. We only saved the features of these images and loaded them in the data loader:

https://github.com/minghangz/ResVG/blob/73c0c02fd2beaa94bfe79229eef5a423631a3036/datasets/dataset.py#L102-L104

The pre-extracted features are also included in the updated link mentioned above.
